src/web/experiments/tests/sign-in-gate-mandatory.ts

```diff
@@ -0,0 +1,30 @@
+import { ABTest } from '@guardian/ab-core';
+
+export const signInGateMandoryTest: ABTest = {
+	id: 'SignInGateMandatory',
+	start: '2021-03-04',
+	expiry: '2021-06-04',
+	author: 'Peter Colley',
+	description:
+		'Compare mandatory gate (an article sigin gate without the dismiss button) with the main signin gate',
+	audience: 0.2,
```

Comment:
We require a sample size of 170k browsers seeing the variant. We'll need to set the audience appropriately to achieve this.

Reply:
Should match what we decide in https://github.com/guardian/frontend/pull/23587